site/docs/spark-configuration.md

```diff
@@ -60,6 +60,7 @@ Both catalogs are configured using properties nested under the catalog name. Com
 | spark.sql.catalog._catalog-name_.uri               | thrift://host:port            | Metastore connect URI; default from `hive-site.xml` |
 | spark.sql.catalog._catalog-name_.warehouse         | hdfs://nn:8020/warehouse/path | Base path for the warehouse directory |
 | spark.sql.catalog._catalog-name_.cache-enabled     | `true` or `false`             | Whether to enable catalog cache, default value is `true` |
+| spark.sql.catalog._catalog-name_.cache.expiration-interval-ms | `30000` (30 seconds)          | Controls the duration for which entries in the catalog are cached; positive values will enable entries to be expired from the catalog cache after that interval if not accessed,`-1` disables cache expiration (same behavior as Iceberg prior to version 0.13.0), and `0` turns off the catalog cache entirely. Default is `30000` (30 sec) ||                                                    |                               |                                                          |
```

Comment:
nit: space before `-1`.

Also I think the config name itself is already pretty self-explanatory, we don't need multiple lines explaining that is cache expiration. We can just point out the important parts which are (1) it's only effective when `cache-enabled` is `true`, and interval equals `0` essentially disables caching.

Reply:
Ok I've updated it to be shorter. Let me know what you think.